core-services/release-controller/_builds/ci-release-images.yaml

```diff
@@ -0,0 +1,733 @@
+##################################################################################
+#                                DO NOT EDIT
+# File generated during execution of: generate-release-controllers.py
+##################################################################################
+
+
+#---------------------------------------------------------------------------------
+# art_input_images.py:149 added the following resource
+#---------------------------------------------------------------------------------
+
+# ART builds many container first images with golang builder images. We want upstream CI
+# builds to use a builder image as close to the ART builder as possible. To that end, 
+# ART will mirror its builder images to api.ci. However, we cannot use these images directly
+# for CI. On top of the ART builders, we need to create yum repositories configuration 
+# files that will allow CI builds access to the same repositories available during an ART
+# build (see https://docs.google.com/document/d/1GqmPMzeZ0CmVZhdKF_Q_TdRqx_63TIRboO4uvqejPAY/edit#heading=h.kv09dibi95bt).
+# 
+# This builder image, now appropriate for CI, needs a few more tools to satsify some 
+# CI testing & packaging tests. A "release" image is therefore built on top of the
+# builder image with these packages installed.
+# 
+# This build creates a RHEL-7 golang release image for CI for OpenShift 4.1.
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  name: rhel-7-release-openshift-4.1-builder
+  namespace: ci
+spec:
+  failedBuildsHistoryLimit: 2
+  output:
+    pushSecret:
+      name: dockerhub
+    to:
+      kind: DockerImage
+      name: docker.io/openshift/origin-release:$rhel-7-release-openshift-4.1
```

Comment:
I don't think we care to do this any longer, can we just push to the ImageStream in `api.ci`?

Reply:
Gladly. Now pushing to openshift:is/release:<tag>